packages/core/docs/performance/other-optimizations.md

```diff
@@ -0,0 +1,105 @@
+# Other optimizations
+
+TODO: Add intro
+
+## Avoid render-blocking resources
+
+Render-blocking resources are scripts, stylesheets, and other imports in the `<head>` that delay the browser from rendering page content to the screen until they are downloaded and parsed. Such resources delay the First Paint - time needed for the browser to render something (i.e., background colors, borders, text, or images) for the first time. To prevent that:
+
+* add the `defer` or `async` attribute to the `<script>`,
+* use `disabled` or `media` attributes for the stylesheets.
+
+TODO: AFAIK, Nuxt.js already adds `defer` or `async` to some scripts. Explain what Nuxt.js already does and how to do the above in Nuxt.js
+
+## Inline critical assets
+
+Consider delivering critical JavaScript/CSS inline and deferring all non-critical assets. You can reduce the size of your pages by only shipping the code and styles that you need at the given time.
+
+Once you've identified critical code, move that code from the render-blocking URL to an inline `<script>` tag. Do the same for the styles required for the first paint inside a `<style>` block at the `<head>` of the HTML page and load the rest of the styles asynchronously using the `preload` link.
+
+TODO: Add Nuxt.js example or libraries that can do that in Nuxt.js
```

Comment:
TODO

Reply:
I would prefer for now to not add those libs, I didn't test any of those yet, so we can add example later.